src/index.ts

```diff
@@ -98,7 +98,7 @@ export default function monacoEditorPlugin(options: IMonacoEditorOpts): Plugin {
             globalAPI: ${globalAPI},
             getWorkerUrl : function (moduleId, label) {
               var result =  paths[label];
-              if (/^((http:)|(https:)|(file:)|(\\/\\/))/.test(result)) {
+              if (/^((http:)|(https:)|(file:)|(\\\\/\\\\/))/.test(result)) {
```

Comment:
are you sure the missing backslashes aren't caused by something else?
for me it works fine with the original code:
![image](https://github.com/vdesjs/vite-plugin-monaco-editor/assets/55899582/a1926e38-337c-4f7e-909c-d922dfaf6698)

Reply:
Yes, this may be a usage issue. I used it in a `amis` project. So I'll close this issue, but will open a new pr to support newest `Vite` version.